docs/erd.md

```diff
@@ -0,0 +1,111 @@
+# Captains Log ERD
+
+```mermaid
+erDiagram
+
+CHANNEL {
+  id sequence
+  project_id int
+  client_id int
+  name text
+  slack_id text
+  inserted_at timestamp
+  updated_at timestamp
+}
+
+CHANNEL ||--|{ CLIENT: "client channels"
+CHANNEL ||--|{ PROJECT: "project channels"
+
+CLIENT {
+    id sequence
+    name text
+    inserted_at timestamp
+    updated_at timestamp
+}
+
+NOTE }|--|| PROJECT: "project notes"
+
+NOTE {
+    id sequence
+    project_id int
+    title text
+    message text
+    persisted boolean
+    inserted_at timestamp
+    updated_at timestamp
+}
+
+PERSISTENCE_PROVIDER {
+    id sequence
+    name text
+    inserted_at timestamp
+    updated_at timestamp
+}
+
+PROJECT ||--|{ PROJECT_PROVIDER: "project providers"
+PERSISTENCE_PROVIDER ||--|{ PROJECT_PROVIDER: "persistence providers"
+
+PROJECT_PROVIDER {
+    id sequence
+    project_id int
+    persistence_provider_id int
+    config jsonb
+    inserted_at timestamp
+    updated_at timestamp
+}
+
+PROJECT {
+    id sequence
+    client_id int
+    name text
+    inserted_at timestamp
+    updated_at timestamp
+}
+
+PROJECT ||--|{ REPOSITORY: "project repos"
+
+REPOSITORY {
+    id sequence
+    project_id int
+    url text
+    name text
+    full_name text
+    observed_id text
+    is_active boolean
+    inserted_at timestamp
+    updated_at timestamp
+}
+
+PERSISTENCE_PROVIDER ||--|{ TOKEN: "provider tokens"
+
+TOKEN {
```

Comment:
Is this used? Or is this what we're using `PROJECT_PROVIDER.config` for?

Reply:
TOKEN was used for the first version of the Confluence OAuth. We have since abandoned plans for this but may revisit in the future.